app/src/main/java/ch/dreipol/rezhycle/fragments/LicenceFragment.kt

```diff
@@ -0,0 +1,33 @@
+package ch.dreipol.rezhycle.fragments
+
+import android.os.Bundle
+import android.view.ContextThemeWrapper
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import ch.dreipol.rezhycle.R
+import com.mikepenz.aboutlibraries.LibsFragmentCompat
+
+open class LicenceFragment : Fragment() {
+
+    private val libsFragmentCompat: LibsFragmentCompat = LibsFragmentCompat()
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        val aboutLibrariesInflater = inflater.cloneInContext(ContextThemeWrapper(inflater.context, R.style.AboutLibrariesTheme))
```

Comment:
Zur Sicherheit: dies ist nur so lange wie der Context lebt? Bzw. wenn man danach wieder zurück navigieren würde auf eine Seite mit einem Button, dann ist der Style wieder zurück gesetzt?

Reply:
Genau, es wird eine Kopie vom context mit einem anderen theme erstellt, der hier nur für das inflaten der aboulibraries view gebraucht wird.
Sonst bleibt der Context mit theme in der ganzen App unverändert